### PR TITLE
[WIP] Allow to configure multiple packages versions on Travis

### LINKED ISSUE #47: Travis: Another versions rules
Make a system to configure another versions rules than Symfony.

This has to be applied on core-bundle with `symfony/security-acl` package.

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Closes #47.

- [x] Rework projects configuration according to `versions` sections
- [x] Adapt `travis.yml.twig`
- [ ] ~~`symfony/security-acl` for core-bundle~~
- [x] `friendsofsymfony/user-bundle` for user-bundle

@core23 AFAIK, you configured Travis on core-bundle to test against some version of `symfony/security-acl`. Did you remember which ones works and fails?

### CODE REVIEW

File: src/Console/Command/DispatchCommand.php [hunk 1]
```diff
@@ -78,6 +78,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->io->warning('This is a dry run execution. No change will be applied here.');
         }
 
+        dump($this->configs);die;
```
Comment (greg0ire) [inline src/Console/Command/DispatchCommand.php]: This is a work in progress :P

Comment (Soullivaneuh) [inline src/Console/Command/DispatchCommand.php]: Yeah, and right now I want to take my beer. :beer: :wink:

File: project/.travis.yml.twig [hunk 2]
```diff
@@ -55,7 +59,9 @@ before_install:
   - if [ "$TARGET" = 'test' ]; then composer self-update --stable; fi;
   - if [ "$TARGET" = 'test' ]; then composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN; fi;
   - if [ "$TARGET" = 'test' ]; then sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json; fi;
-  - if [ "$TARGET" = 'test' -a "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+{% for package_name,package_versions in versions if package_versions|length > 0 %}
+  - if [ "$TARGET" = 'test' -a "${{ package_name|upper }}" != "" ]; then composer require "{{ packages[package_name] }}:${{ package_name|upper }}" --no-update; fi;
```
Comment (greg0ire) [inline project/.travis.yml.twig:63]: @OskarStark I think the `packagen_name|upper` is the reason

Comment (OskarStark) [reply project/.travis.yml.twig:63]: Jepp, that seems to be correct, but is not needed anymore with GithubActions

Comment (OskarStark) [reply project/.travis.yml.twig:63]: For reference:
https://github.com/sonata-project/SonataAdminBundle/blob/7ca2b6e800873159d3ba23bdff613be096dac4b4/.github/workflows/test.yaml#L22-L47

**Pull request merged**